packages/@react-spectrum/overlays/src/Popover.tsx

```diff
@@ -78,8 +81,9 @@ const PopoverWrapper = forwardRef((props: PopoverWrapperProps, ref: RefObject<HT
     maxHeight: null
   }, state);
 
+  // Attach Transition's nodeRef to outer most wrapper for node.reflow: https://github.com/reactjs/react-transition-group/blob/c89f807067b32eea6f68fd6c622190d88ced82e2/src/Transition.js#L231
   return (
-    <>
+    <div ref={wrapperRef}>
```

Comment:
We could instead attach the nodeRef to the Underlay, however it doesn't exist in the `isNonModal` case with popover. Adding a wrapping div should be safe because Underlay and the overlay (Popover, Tray, Modal) all use `position: fixed` or `position: absolute`